kalite/distributed/features/steps/login.py

```diff
@@ -33,7 +33,14 @@ def step_impl(context):
 
 @then("there should be no facility drop down")
 def step_impl(context):
-    assert not find_id_with_wait(context, "id_facility-container")
+    try:
+        # TODO(benjaoming): This is a crazy test... waiting for a timeout in
```

Comment:
Oh but I changed that :) ...because it was cause illogical test failures elsewhere... if a timeout occurs, you'd want to know and not have the test continue with a None value